src/content/team/team.json

```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Zefeng (Daniel) Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Boolin | Dolphin of the East"
```

Comment:
<img width="755" alt="Screen Shot 2022-04-24 at 12 30 11 AM" src="https://user-images.githubusercontent.com/25839028/164956724-14931a3e-ea63-4167-be75-98725176383a.png">

Reply:
so true king

Reply:
Please check your spelling and rerequest